packages/xo-web/src/common/editable/index.js

```diff
@@ -341,74 +332,91 @@ export class Number extends Component {
   }
 }
 
-export class Select extends Editable {
+class SimpleSelect_ extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
-    renderer: PropTypes.func,
-  }
-
-  componentWillReceiveProps(props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
+    optionRenderer: PropTypes.func,
+    value: PropTypes.oneOfType([PropTypes.oneOf([null]), PropTypes.object]),
   }
 
   get value() {
-    return this.props.options[this.state.valueKey]
-  }
-
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
+    return this.state.value
```

Comment:
What if the user never changed the value himself? Then it should come from the props.